### ISSUE TITLE
SVG support?

### ISSUE THREAD

**Issue by strarsis** [nicolas-t/Chocolat#77]
Are also SVGs supported? A note in the README would be nice.

**Comment by nicolas-t** [comment]
What do you mean ? SVG as images ? I haven't tried to be honest.

**Comment by strarsis** [comment]
@nicolas-t: Right, adding SVGs usually requires a bit of extra CSS adjustments to prevent it collapsing or distorting and making it fit in the lightbox.
